src/nvim/main.c

```diff
@@ -1878,10 +1878,8 @@ static void source_startup_scripts(mparm_T *parmp)
   TIME_MSG("sourcing vimrc file(s)");
 }
 
-/*
- * Setup to start using the GUI.  Exit with an error when not available.
- */
-static void main_start_gui(void)
+/// Informs the user that Nvim has no built-in GUI, then exits.
+static void gui_noop(void)
```

Comment:
I think it's too early to change this at present. We haven't decided what to do with the `-g` option.